src/commands/workers.rs

```diff
@@ -64,6 +68,13 @@ enum Subcommand {
         /// The ID of the worker to announce exit.
         id: String,
     },
+
+    /// Update a worker.
+    Update {
```

Comment:
If the worker must already exist at this moment (there will be always an error when ID is not present in worker spec file), does it make sense to add required argument `id` here? Just like in `Subcommand::Get`